hal/src/mrc/include/mrc/NetComm.h

```diff
@@ -83,21 +123,25 @@ struct JoystickPovs {
   size_t GetCount() const { return Count; }
 
  private:
-  std::array<uint8_t, MRC_MAX_NUM_POVS> PovsStore;
+  std::array<uint8_t, MRC_MAX_NUM_POVS> PovsStore{};
   uint8_t Count{0};
 };
 
 struct JoystickButtons {
-  uint64_t Buttons;
+  uint64_t Buttons{0};
+  uint64_t AvailableButtons{0};
 
-  void SetCount(uint8_t NewCount) {
-    Count = (std::min)(NewCount, static_cast<uint8_t>(MRC_MAX_NUM_BUTTONS));
+  void SetMaxAvailableToCount(size_t Count) {
```

Comment:
```suggestion
  void SetMaxAvailableCount(size_t Count) {
```
Same comment as `JoystickAxes`, resolve if the `To` is intentional.